story/ch1/zonik_001_vm0x_n01.txt

```diff
@@ -1,23 +1,23 @@
-`Mion then lowered her hand to about 1 meter above the ground.`
-`"But you've only known me for a month..."`
-`"You've grown up so magnificently...`
-` All the trouble this old man had to go through was worth it."`
-`She gently took out a handkerchief and pretended to weep into it.`
-`Just what is she trying to pull off at 8 o'clock in the morning...?`
-`"Wasn't it me who went through the trouble there?!"`
-`"You've grown up so much, you even have a beard~!"`
-`"My beard hasn't started growing yet!"`
-`"No, it's already growing. You got yourself a razor for it last week, right?"`
-`"That was my father's... Wait, how do you know that?"`
-`Mion sneered at the question, and then wagged her index finger right in my face.`
-`...Just what is she playing at?`
-`"Hahaha, NEVER underestimate Mion-san's intelligence gathering skills!`
-` From your last meal to Rena's measurements, this old man knows it all."`
-`My interest piqued, I placed my hand on her shoulder while she was still smirking.`
-`"...I'll bite, what are Rena's measurements?"`
-`"Just as I thought, you really want to knoooow?`
-` By the way, it's expensive~ How much would you pay?"`
-`Catching on to our awful discussion, Rena became flustered and began to ramble in a near panic.`
-`"Don't you ignore me too, Rena! I'll put out a village-wide circular saying how you two are madly in love!"`
-`"Keiichi-san's lunch box is most assuredly a plain, miserly rice and seaweed~!`
-` Why don't you just show it to us? Come now!"`
+`Ardından Mion, elini yerden yaklaşık 1 metre yüksekliğe indirdi.`
+`"Ama beni sadece bir aydır tanıyorsun..."`
+`"Çok harika bir şekilde büyümüşsün...`
+` Bu yaşlı adamın çektiği tüm sıkıntılara değmiş."`
+`Yavaşça bir mendil çıkardı ve ağlar gibi yaptı.`
+`Sabahın sekizinde ne yapmaya çalışıyor...?`
+`"O kadar zahmete giren ben değil miydim?!"`
+`"Çok büyümüşsün, sakalın bile var~!"`
+`"Sakalım çıkmaya başlamadı bile!"`
+`"Hayır, zaten çıkıyor. Geçen hafta kendine bir jilet almıştın, değil mi?"`
+`"O babamındı. Bir dakika, sen bunu nereden biliyorsun?"`
+`Mion bu soruya burun kıvırdı ve işaret parmağını yüzüme doğru salladı.`
```

Comment:
and then wagged her index finger right in my face (işaret parmağını hayır dercesine gözümün önünde salladı)